fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -425,8 +449,16 @@ private enum class Highlight(
 private fun ProductRecommendation(
     product: RecommendedProductState.Product,
     onClick: () -> Unit,
+    onImpression: (String) -> Unit,
 ) {
-    Column(verticalArrangement = Arrangement.spacedBy(8.dp)) {
+    Column(
+        modifier = Modifier.onShown(
+            threshold = 0.5f,
```

Comment:
Let's extract this as a constant, Is 0.5 confirmed?

Reply:
It's what's been used for Pocket, but I can double check for here

Reply:
Thanks!

Reply:
Finally got confirmation that 50% is good for this impression